salt/cloud/libcloudfuncs.py

```diff
@@ -336,7 +341,7 @@ def reboot(name, conn=None):
         __utils__["cloud.fire_event"](
             "event",
             "{} has been rebooted".format(name),
-            "salt-cloud" "salt/cloud/{}/rebooting".format(name),
+            "salt/cloud/{}/rebooting".format(name),
```

Comment:
Is this change right?  It looks like it's changing the event message not just the formatting?

Reply:
yep, it's correct